test/metabase/dashboard_subscription_test.clj

```diff
@@ -634,12 +635,15 @@
       [Dashboard     {dashboard-id :id
                       :as dashboard}   {:name "Dashboard"}
        DashboardCard _                 {:dashboard_id           dashboard-id
-                                        :visualization_settings {:text "Markdown"}}
+                                        :visualization_settings {:text "Markdown"}
```

Comment:
This is the cause of https://github.com/metabase/metabase/actions/runs/4343640011/jobs/7585948716#step:4:460.

Since these dash cards are of the type, we need an explicit row to make the comparator consistent.

Reply:
> these dash cards are of the type

I don't understand this sentence.